### PR TITLE
add a blurb about waivers to compliance reports docs

### PR DESCRIPTION
**Pull request by vjeffrey**
Signed-off-by: Victoria Jeffrey <vjeffrey@chef.io>

### :nut_and_bolt: Description: What code changed, and why?
just adding a blurb about waivers to the compliance docs

### :athletic_shoe: How to Build and Test the Change
```
cd components/automate-chef-io
make serve
```

### CODE REVIEW

File: components/automate-chef-io/content/docs/reports.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ Note: The count of controls listed in the _Profile Status_ pane of the Reports O
 
 ### Dates in Compliance Reports
 
-The dashboard shows the results of all scans with end times on the _currently selected day_. The selected day is highlighted in the search bar.
+The dashboard shows the results of all scans with end times on the _currently selected day_. The selected day is highlighted in the search bar, and is in UTC.
```
Comment (mjingle) [inline components/automate-chef-io/content/docs/reports.md:26]: @vjeffrey UTC is Coordinated Universal Time and applies to time zones. Are you talking about the date format?

File: components/automate-chef-io/content/docs/reports.md [hunk 2]
```diff
@@ -72,6 +72,11 @@ Please note the only supported filters for deep filtering are:
     * one profile
     * one profile and one of its child controls
 
+### Waivers
+
+Waived status of nodes can be seen throughout the pages where status is listed. Users can hover over waived controls' status icon in the controls list to view more details about the waiver applied to the control.
```
Comment (mjingle) [inline components/automate-chef-io/content/docs/reports.md:77]: @vjeffrey When you describe the "waived controls' status icon in the controls list", does that refer to the Node Status column in the Controls list (/compliance/reports/controls) OR are you talking about the Waived Nodes filter in the Node list (compliance/reports/nodes)? Both lists are located under Compliance Reports.

Comment (vjeffrey) [reply components/automate-chef-io/content/docs/reports.md]: Node Status column in the Controls list

File: components/automate-chef-io/content/docs/reports.md (hunk 1 above)
Comment (vjeffrey) [reply components/automate-chef-io/content/docs/reports.md]: yupp

File: components/automate-chef-io/content/docs/reports.md [hunk 3]
```diff
@@ -74,8 +76,11 @@ Please note the only supported filters for deep filtering are:
 
 ### Waivers
 
-Waived status of nodes can be seen throughout the pages where status is listed. Users can hover over waived controls' status icon in the controls list to view more details about the waiver applied to the control.
-Use of waivers is configured using Chef InSpec: https://www.inspec.io/docs/reference/waivers/
+A node's waived status appears if applicable in displays where a node's status appears in Chef Automate. 
+The Compliance Reports overview displays the node count and history of waived nodes, and the count and history of waived controls.
+Hover over the control's Waived icon in the Controls list to view more details about the waiver applied to the control.
```
Comment (vjeffrey) [inline components/automate-chef-io/content/docs/reports.md:81]: should we say "controls list view's node status column" ?

Comment (vjeffrey) [reply components/automate-chef-io/content/docs/reports.md:81]: @mjingle ^^

Comment (mjingle) [reply components/automate-chef-io/content/docs/reports.md:81]: @vjeffrey The icon technically appears for each individual control in Controls, but thank you for making me realize this paragraph needed further expansion!

**Pull request merged**